OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -76,6 +76,9 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Vertical rate of turn.")]
 		public readonly int VerticalRateOfTurn = 6;
 
+		[Desc("Gravity applied while in free fall.")]
```

Comment:
> Could you please add that the negative of this value is used internally?

Personally, I think modders don't need to know how this works internally. In fact I'm afraid adding that to the description might just cause unnecessary confusion.